dlt/common/pipeline.py

```diff
@@ -54,7 +54,7 @@ def asdict(self) -> DictStrAny:
         """A dictionary representation of NormalizeInfo that can be loaded with `dlt`"""
         d = self._asdict()
         # list representation creates a nice table
-        d["row_counts"] = [(k, v) for k, v in self.row_counts.items()]
+        d["row_counts"] = [{"table_name": k, "count": v} for k, v in self.row_counts.items()]
```

Comment:
this makes a much nicer table..

Reply:
absolutely! this will get a proper treatment in #757